conda-recipes/llvmlite/conda_build_config.yaml

```diff
@@ -1,14 +1,14 @@
 # Numba/llvmlite stack needs an older compiler for backwards compatability.
-c_compiler_version:         # [linux and osx]
+c_compiler_version:
   - 7                       # [linux and (x86_64 or ppc64le)]
   - 9                       # [linux and aarch64]
   - 10                      # [osx]
 
-cxx_compiler_version:       # [linux and osx]
+cxx_compiler_version:
   - 7                       # [linux and (x86_64 or ppc64le)]
   - 9                       # [linux and aarch64]
   - 10                      # [osx]
```

Comment:
you should use here instead (osx and x86_64), as for osx-arm64 there is no clang 10